src/index/import/markdown.ts

```diff
@@ -116,7 +116,7 @@ export function markdownImport(
     for (const block of blocks.values() as Iterable<JsonMarkdownBlock>) {
         const section = sections.getPairOrNextLower(block.$position.start);
 
-        if (section && section[1].$position.end >= block.$position.end) {
+        if (section /* && section[1].$position.end >= block.$position.end */) {
```

Comment:
Commenting out code is fine for debugging but I don't recommend it when merging; I'd just delete the commented out code.